### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Jerzus** [lukzielinski/Disnay-characters#1]
add license

**Comment by stefan-hdt** [comment]
Yes, a license statement in the README file and LICENSE file with the license text and copyright statement would be greatly appreciated. Otherwise, there are legal troubles using the project.
